cmd/soroban-cli/src/commands/contract/build.rs

```diff
@@ -295,8 +299,54 @@ impl Cmd {
             return Ok(());
         }
 
-        let mut wasm_bytes = fs::read(target_file_path).map_err(Error::ReadingWasmFile)?;
+        // get existing wasm bytes
+        let wasm_bytes = fs::read(target_file_path).map_err(Error::ReadingWasmFile)?;
+        let existing_meta: Vec<ScMetaEntry> = Spec::new(&wasm_bytes).unwrap().meta;
```

Comment:
Why do we use the `Spec` class to read the meta out of the `wasm_bytes`? I would expect that we wouldn't need to parse the existing meta. The new meta can be encoded to XDR on its own and appended to the existing meta's bytes without decoding the existing meta.